plugins/woocommerce-blocks/tests/e2e/tests/reviews-by-product/reviews-by-product.block_theme.spec.ts

```diff
@@ -44,21 +40,51 @@ test.describe( `${ blockData.name } Block`, () => {
 		).toBeVisible();
 	} );
 
-	test( 'can change sort order in the frontend', async ( {
+	test( 'sorts by most recent by default and can sort by highest rating', async ( {
 		page,
 		frontendUtils,
+		editorUtils,
 	} ) => {
-		await page.goto( '/reviews-by-product/' );
+		await editorUtils.publishAndVisitPost();
+		const block = await frontendUtils.getBlockByName( BLOCK_NAME );
 
-		const block = await frontendUtils.getBlockByName( blockData.name );
-		let firstReview;
-		firstReview = block.locator( blockData.selectors.frontend.firstReview );
-		await expect( firstReview ).toHaveText( hoodieReviews[ 1 ].review );
+		const reviews = block.locator(
+			'.wc-block-components-review-list-item__text'
+		);
+
+		await expect( reviews.nth( 0 ) ).toHaveText( latestReview.review );
 
 		const select = page.getByLabel( 'Order by' );
 		select.selectOption( 'Highest rating' );
 
-		firstReview = block.locator( blockData.selectors.frontend.firstReview );
-		await expect( firstReview ).toHaveText( hoodieReviews[ 0 ].review );
+		const highestRating = allReviews.sort(
+			( a, b ) => b.rating - a.rating
+		)[ 0 ];
+
+		await expect( reviews.nth( 0 ) ).toHaveText( highestRating.review );
+	} );
+
+	test( 'can sort by lowest rating', async ( {
+		page,
+		frontendUtils,
+		editorUtils,
+	} ) => {
+		await editorUtils.publishAndVisitPost();
+		const block = await frontendUtils.getBlockByName( BLOCK_NAME );
+
+		const reviews = block.locator(
+			'.wc-block-components-review-list-item__text'
+		);
+
+		await expect( reviews.nth( 0 ) ).toHaveText( latestReview.review );
+
+		const select = page.getByLabel( 'Order by' );
+		select.selectOption( 'Lowest rating' );
```

Comment:
```suggestion
		await select.selectOption( 'Lowest rating' );
```

Reply:
Thank you for spotting this 😓 fixed here https://github.com/woocommerce/woocommerce/pull/44880/commits/40cabeabe5fd2f2a3f4bd338ace15715fe69d726